src/script/engines/lua.c

```diff
@@ -1519,3 +1528,35 @@ static int _luaRequireShim(lua_State* lua) {
 	int newtop = lua_gettop(luaContext->lua);
 	return newtop - oldtop + 1;
 }
+
+static int _luaPrintShim(lua_State* lua) {
+	int n = lua_gettop(lua);
+
+	lua_getglobal(lua, "console");
+	lua_insert(lua, 1);
+
+	// The first upvalue is either "log" or "warn"
+	lua_getglobal(lua, "console");
+	lua_pushvalue(lua, lua_upvalueindex(1));
+	lua_gettable(lua, -2);
+
+	lua_insert(lua, 1);
+	lua_pop(lua, 1);
+
+	// If console:log is variadic and stringifies by itself:
```

Comment:
```suggestion
	// TODO when console:log is variadic and stringifies by itself:
```
I actually do search the codebase for `TODO` sometimes.